Server/API/rooms/join/join.go

```diff
@@ -2,8 +2,9 @@ package join
 
 import (
 	"github.com/labstack/echo/v4"
+	entities "ishtaloo.io/Entities"
 )
 
-func JoinController(e *echo.Echo) {
-	e.POST("/rooms/join/:roomId", postJoin)
+func JoinController(e *echo.Echo, sseChannel *entities.SSEChannel) {
```

Comment:
can sse channel be a global var? does it have to be directly inherited?

Reply:
I can't export it to other packages because it creates an import loop.
I couldn't find a way to do it differently

Reply:
OK